app/controllers/idv/in_person/public/usps_locations_controller.rb

```diff
@@ -3,6 +3,7 @@ module InPerson
     module Public
       class UspsLocationsController < ApplicationController
         include RenderConditionConcern
+        include UspsInPersonProofing
```

Comment:
is this being included to use methods, or to shorten the namespace for things like `EnrollmentHelper`? IMO it's more confusing to use it that way, easier to just use the long names